### ISSUE TITLE
build_env_bionet fails when config_file is not explicitly provided

### ISSUE THREAD

**Issue by HDictus** [AllenInstitute/bmtk#162]
The default value of `None` for the kwarg `config_file` to `bmtk.utils.sim_setup.build_env_filternet` is an invalid one and causes an error, since a string or Path is expected. I suggest only passing config_file to `env_builder.build` if it is not `None`, so that env_builder can specify its own correct default.

**Comment by mwstroud** [comment]
This is true for build_env_bionet as well. If looking for quick solution:
In build_env_bionet( ), add the parameter: config_file = 'config.json' (as noted above)
Then, when running the  simulation,:
bionet.Config.from_json() now wants the input to be 'config.json' instead of 'simulation_config.json'

Is this an update issue? It has been only recent that I could not build (before this solution).
